docker-compose.yml

```diff
@@ -354,7 +354,7 @@ services:
               source: postgres-data
               target: /var/lib/postgresql/data
         ports:
-            - "${POSTGRES_PORT}:5432"
+            - "127.0.0.1:${POSTGRES_PORT}:5432"
```

Comment:
we also want this to be exposed to the network so we can monitor the progress from DSD